recipes/nifti_clib/LICENSE.txt

```diff
@@ -0,0 +1,9 @@
+Niftilib has been developed by members of the NIFTI DFWG and volunteers in the
```

Comment:
Looks like there is a file named LICENSE in the tarball, so you can remove this one, and have `license_file: LICENSE` instead.